src/network/gamehost.cc

```diff
@@ -1935,7 +1935,7 @@ bool GameHost::client_may_change_speed(uint8_t playernum) const {
 		return false;
 	}
 	const Widelands::PlayerEndStatus* pes =
-	   d->game->player_manager()->get_player_end_status(playernum + 1);
+	   d->game->player_manager()->get_player_end_status(playernum);
```

Comment:
Yes, tested and restores correct behaviour.

```suggestion
	   d->game->player_manager()->get_player_end_status(playernum + 1);
```